### ISSUE TITLE
please update to python 3 and use gpio zero so the Libreelec users can use your hardware

### ISSUE THREAD

**Issue by CBDesignS** [Argon40Tech/libreelec_package_argonforty-device#1]
Not every purchaser of your hardware (Argon One V3) will use a destop variant of ubuntu/rasberry pi os etc and I think quite a few will use it only as a media player running Libreelec but these files are older than time. python 2 has been depreciated for many many years as has rpi.gpio. when the new rpi hardware chaged the old way of using gpio was also left behind. This repo is at least 2 years out of date code wise and the last updates were 3 years ago.

this Libreelec addon needs some serious rewriting to bring it up to the current standards.

Python 3 update since python 2 is long dead

gpio zero update since rpi.gpio is long dead

**Comment by Renato-4132** [comment]
https://github.com/HungerHa/libreelec_package_argonforty-device/tree/libreelec12

**Comment by CBDesignS** [comment]
I have used Harrys code since he started rewriting it, because the maker of the hardware could not be bothered to update their own code, as it is easier to rely on others to fix their products and claim its "open source" so YOU want it working then YOU fix it..

Big thanks to Harry H for his work and all those that contributed to it.

**Comment by Renato-4132** [comment]
Yes, a big thanks , bit the script is only kodi and libreelec .
What about the other dists.
Argon should spend some time to fix this situation or will loose clients. It's amazing they don't understand that.

**Comment by Renato-4132** [comment]
Also everybody is claiming this situation on the argon main site, with no answer. 
For how much the argon costs ,and nvme compatibily problems , it's no si worth the money

**Comment by CBDesignS** [comment]
that is true but they will just keep on pumping out hardware with basic support and rely on others to help out, just like most of the 3D printer control boards from China. . The price point for the case and other parts are a lot cheaper than the rpi 5 itself. Also the Rpi is a development board aimed at the user who wants to develop and learn/can write code etc.. I also suspect that for every case sold there are another 2 people wanting one so they will not loose clients as they only produce them and release stock to keep the market wanting more, just like the way xbox and playstation do at new console launch time. "keep the market hungry and the will want more and pay more"
